recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -49,9 +49,18 @@ def configure(self):
         if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
             raise ConanInvalidConfiguration("gcc < 5 is not supported")
 
+    @property
+    def _get_compatible_spirv_tools_version(self):
+        return {
+            "1.2.182": "2021.2",
+            "1.2.154.0": "2020.5",
+        }.get(str(self.version), False)
+
     def requirements(self):
-        self.requires("spirv-tools/2020.5", private=True)
+        self.requires("spirv-tools/{}".format(self._get_compatible_spirv_tools_version), private=True)
         self.requires("vulkan-headers/{}".format(self.version))
+        if tools.Version(self.version) > "1.2.154.0":
+            self.requires("robin-hood-hashing/3.11.1")
```

Comment:
In which version exactly this dependency was added (in case someone would like to add an intermediate version)?

Reply:
The commit involved is just before release 1.2.173 (commit is on 19 Mar, release on 24 Mar)